erpnext/setup/setup_wizard/operations/install_fixtures.py

```diff
@@ -30,6 +30,8 @@ def install(country=None):
 		{'doctype': 'Domain', 'domain': 'Agriculture'},
 		{'doctype': 'Domain', 'domain': 'Non Profit'},
 		{'doctype': 'Domain', 'domain': 'Insight Engine'},
+		{'doctype': 'Domain', 'domain': 'Cannabis'},
```

Comment:
- Are we not moving Cannabis related fields to custom fields ??
- eg: If I am not a cannabis user, I dont want to see Package Tag field anywhere